sycl/source/detail/platform_impl.cpp

```diff
@@ -494,6 +496,18 @@ void platform_impl::getDevicesImplHelper(ur_device_type_t UrDeviceType,
   // handles to do a cleanup later
   std::vector<ur_device_handle_t> UrDevicesToCleanUp = UrDevices;
 
+  // Filter out FPGA devices since they are no longer supported.
```

Comment:
Note for reviewers: Currently I am filtering out all UR_DEVICE_TYPE_FPGA  returned by UR API.
I.e. we just ignore them and don't return from `platform::get_devices` and everywhere else.

Reply:
Thanks, removed this code.